src/Sign.SignatureProviders.KeyVault/KeyVaultService.cs

```diff
@@ -48,50 +51,61 @@ public void Dispose()
 
         public async Task<X509Certificate2> GetCertificateAsync(CancellationToken cancellationToken)
         {
-            KeyVaultCertificateWithPolicy certificateWithPolicy = await GetCertificateWithPolicyAsync(cancellationToken);
+            KeyVaultCertificate certificateWithPolicy = await GetCertificateVersionAsync(cancellationToken);
```

Comment:
`certificateWithPolicy` should probably also be renamed here and in other spots.

Reply:
Renamed the relevant variables